packages/web-components/fast-components/src/slider/slider.styles.ts

```diff
@@ -82,7 +82,7 @@ export const SliderStyles = css`
         transform: translateY(calc(var(--thumb-translate) * 1px));
     }
     :host(.horizontal) {
-        min-width: calc(var(--design-unit) * 60px);
+        min-width: calc(var(--design-unit) * 1px);
```

Comment:
This works, or rather doesn't break any of our fixtures, but I'm wondering now does it make sense to set the min-width to something smaller than what the thumb is going to be set to? which is calc(var(--thumb-size) * 1px), typically I think this is 16px, for DesignSystemDefaults at least I think.

Reply:
I am totally agreed so the code for `min-width` will be:

```suggestion
        min-width: calc(var(--thumb-size) * 1px);
```

right? 

Also, as I mentioned in the **findings** section of the pull request what can we do with the vertical one?

Reply:
I read your findings there about the vertical part, if you could I think it would be good to create an issue for that and maybe link back to this one, and we can look at it again.

Reply:
Good idea, I am goint to do that thanks.

Reply:
that looks good to me